lib/Elevate/Components/MySQL.pm

```diff
@@ -72,6 +72,17 @@ sub _reinstall_mysql_packages {
     INFO("Restoring $cnf_file.rpmsave_pre_elevate to $cnf_file...");
     File::Copy::copy( "$cnf_file.rpmsave_pre_elevate", $cnf_file ) or WARN("Couldn't restore $cnf_file.rpmsave: $!");
 
+    # Apparently the APITool is in fact unable to figure out that --output=json means I want JSON in this context,
+    # so just load the modules directly and do what the API does to gather the data I want.
+    require Cpanel::Services::Enabled;
+    my $enabled = Cpanel::Services::Enabled::is_enabled('mysql');
```

Comment:
let's store that status to the elevate status file so we can reuse that state later when reinstalling/restarting MySQL and do not block on it when disabled